src/components/Controls/single-controls/LangControl/LangControl.scss

```diff
@@ -12,7 +12,7 @@
     }
 
     &__popup-tooltip {
-        padding: 0;
+        --g-popover-padding: 0px;
```

Comment:
instead of redefining the variable, we cannot raise the specificity of the class and leave the redefinition of the padding ?